pkgs/development/python-modules/anthropic/default.nix

```diff
@@ -92,6 +96,8 @@ buildPythonPackage rec {
     # Test require network access
     "tests/api_resources"
     "tests/lib/test_bedrock.py"
+    # TODO remove once inline-snapshot >= 0.28 is available
```

Comment:
What's failing that needs that version of that dependency? Can the relevant fix be backported (unless it's a feature that's missing in which case why is it depending on unreleased features)?

Reply:
It's looking for an import that was introduced in inline-snapshot 0.28. That update is in staging already while master is at 0.24. I was testing against master before moving this to staging. I could drop this now.

Reply:
Line removed.